libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.ts

```diff
@@ -173,6 +173,9 @@ export class MembersDynamicListComponent implements AfterViewInit, OnInit, OnCha
   changeStatus(event: Event, member: RichMember, groupId?: number): void {
     event.stopPropagation();
 
+    // Do not open status dialog, if caller isn't VOADMIN/PERUNADMIN
+    if (!this.authResolver.isThisVoAdmin(this.voId)) return;
```

Comment:
If possible please do this check with the `isAuthorized` method from `GuiAuthResolver`.

Reply:
We have to make sure the correct policy is used for vo/group based on which page is the table used.